.huskyrc

```diff
@@ -3,6 +3,6 @@
   "//2": "Use git push --no-verify to bypass the pre-push hook",
   "hooks": {
     "pre-commit": "npm run precommit",
-    "pre-push": "npm run lint"
+    "pre-push": "npm run lint -- --fix"
```

Comment:
We don't want to use fix here because it won't have any effect. The flow of events will look like this:

```
git commit
git push
pre-push hook runs and modifies a file
push executes
modified file remains, needing to be added to a separate commit.
```

Reply:
Oh, right. 

I removed it in my next commit of .huskyrc though.
I will be more careful while I make another PR :D